adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -98,4 +100,47 @@ public Route addRoute(@NotNull final Route newRoute) throws IllegalArgumentExcep
     public Route getRouteById(final String routeId) {
         return routeCollection.get(routeId);
     }
+
+    /**
+     * Add a Transfer representing a row from transfers.txt to this. Return the entity added to the repository if the
```

Comment:
I think there is a word missing at the end of the first sentence here - `transfers.txt to this`